spaceflights-pandas-viz/{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -3,9 +3,23 @@ requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
 [project]
+requires-python = ">=3.9"
```

Comment:
✔️

Reply:
Shall we also update `requirements.txt` for consistency since requires-python = ">=3.9"?